.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-windows`
ConsoleHost unit tests.ApartmentState WPF tests.WPF requires STA and will work
```powershell
The following exception occurred while retrieving member "Load": "Could not load type 'MS.Internal.SecurityCriticalDataForSet`1' from assembly 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'."
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 733
733:             $Window = [System.Windows.Markup.XamlReader]::Load($reader)

```

Reply:
Probably sporadic failure, will make empty commit to retry (did not see this failure in my own Azure Pipelines builds)